device-side-launch/dsl.c

```diff
@@ -78,7 +78,7 @@ CUdevice cudaDeviceInit()
 
     checkCudaErrors( cuDeviceComputeCapability(&major, &minor, cuDevice) );
     printf("compute capability = %d.%d\n", major, minor);
-    if (major < 3 || minor < 5) {      
+    if (major < 3 && minor < 5) {
```

Comment:
I don't think this is right. Maybe we should just check for SM major version at least 5? Kepler is already deprecated.